tests/unit/modules/dcnm/test_dcnm_intf.py

```diff
@@ -516,15 +494,6 @@ def load_svi_fixtures(self):
             self.run_dcnm_send.side_effect = [
                 self.playbook_mock_vpc_resp,
                 playbook_svi_intf1,
-                self.playbook_mock_succ_resp,
```

Comment:
Why are these being removed?

Reply:
They are some extra mock data that was added which is not actually required. I have cleaned up some of these and kept only those that are required